FluidNC/src/xmodem.cpp

```diff
@@ -129,13 +101,13 @@ static void flushinput(void) {
 static bool    held = false;
 static uint8_t held_packet[1024];
 static void    flush_packet(size_t packet_len, size_t& total_len) {
-    if (held) {
-        // Remove trailing ctrl-z's on the final packet
+       if (held) {
```

Comment:
Maybe your autoformat is not using clang-format?  If it is using clang-format, then the .clang-format rules in the repo should, in principle, govern the styling.

Reply:
Maybe this will help <https://code.visualstudio.com/docs/cpp/cpp-ide> .  In principle, if you have the C++ extensions enabled, you get clang-format, but maybe you have some other extensions that use a different formatter, or there is some other setting override.  Vscode is pretty much infinitely customizable, which means that no two people have a similar experience.